src/Bielu.Examine.ElasticSearch/Services/PropertyMappingService.cs

```diff
@@ -28,10 +27,11 @@ protected virtual void FromExamineType(ref PropertiesDescriptor<BieluExamineDocu
             "float" => descriptor.FloatNumber(fieldName),
             "long" => descriptor.LongNumber(fieldName),
             var type when _integerFormats.Contains(type) => descriptor.IntegerNumber(fieldName),
-            "raw" => descriptor.Keyword(fieldName),
+            "raw" => descriptor.Text(fieldName, configure => configure.Analyzer("keyword")),
```

Comment:
All raw fields were indexed as keyword. I changed this to text with keyword analyzer. Not entirely sure if this is correct.